english/light-fire/magic-world/follow/dream/fight/sword/lazarus/gods/god.txt

```diff
@@ -0,0 +1 @@
+Welcome to the brave new world of Gods and Monsters.
```

Comment:
Please make sure you name all of your files with the extension `.md`.  Otherwise, the files will not display correctly on GitHub.